Tutorial.md

```diff
@@ -0,0 +1,66 @@
+# MSON Tutorial
+
+MSON is short for Markdown Syntax for Object Notation, and it's a way to represent data structures in a human readable plain text form. This tutorial will take you though the basics of MSON and how you can use MSON to describe a data structure.
+
+> **NOTE:** **Additional MSON Resources**
+>
+> + [MSON Specification](https://github.com/apiaryio/mson/blob/master/MSON%20Specification.md)
+> + [Examples](https://github.com/apiaryio/mson#notational-conventions)
+
+## Data structures in MSON
+
+To get started, we're going to create a data structure in MSON to represent a Person.
+
+A data structure to store a Person could be represented in MSON using the following:
+
+```apib
+# Person (object)
```

Comment:
No, it is not a bug. It is a smart codebase (and smart spec too) :smile: 

* Default for a member without nested members is `string`
  * If it has nested members, then `object`
* Default for named types is `object`. `Attributes` is a named type.